third_party/bls-signatures/install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+SCRIPT_PATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+BUILD_PATH="$SCRIPT_PATH/build"
+
+if [ ! -d $BUILD_PATH ]; then
+	echo "$BUILD_PATH doesn't exist. Run \"make build-bls\" first." >/dev/stderr
+	exit 1
+fi
+
+# Install the library
+sudo make -C $BUILD_PATH install
```

Comment:
Shall we move `sudo` to the tenderdash make file to make it more visible?

Reply:
f7f7361, why not :)